FormFactory/Attributes/RegularExpressionAttribute.cs

```diff
@@ -10,10 +10,10 @@ public RegularExpressionAttribute(string pattern)
 
         public string FormatErrorMessage(string propertyVmDisplayName)
         {
-            return $"{propertyVmDisplayName} must be ${FriendlyFormat ?? $"match pattern '{Pattern}'"}";
+            return $"{propertyVmDisplayName} must {FriendlyFormat ? "be " + FriendlyFormat : "match pattern '{Pattern}'"}";
```

Comment:
Got the same issue. Here's the fix:
`return $"{propertyVmDisplayName} must be {FriendlyFormat ?? $"match pattern '{Pattern}'"}";`

I only removed the dollar sign after "must be".
Hope to see this in the next release soon 😊